README.md

```diff
@@ -162,7 +162,7 @@ This integer specifies the LUKS key size (in bits).
 This integer specifies the LUKS version to use.
 
 ##### `deduplication`
-This specifies whether or not the Virtual Data Optimizer (VDO) will be used.
+This specifies whether the Virtual Data Optimizer (VDO) will be used.
```

Comment:
I would prefer "This specifies that the pool . . ." - I prefer indicative rather than conditional or subjunctive

Reply:
I guess the same https://github.com/linux-system-roles/storage/pull/230#discussion_r716107461 applies here - so whether is more appropriate - but I'm not sure what happens if you specify `deduplication: false` on an existing volume/pool that is using VDO deduplication . . .